openbas-model/src/main/java/io/openbas/database/model/Team.java

```diff
@@ -134,6 +148,7 @@ public long getUsersNumber() {
   @ArraySchema(schema = @Schema(type = "string"))
   @JsonProperty("team_exercise_injects")
   @JsonSerialize(using = MultiIdListDeserializer.class)
+  @Schema(description = "List of injects from all simulations of the team")
   public List<Inject> getExercisesInjects() {
```

Comment:
With a multiIdListDeserializer, this is only the list of ID's on client side, no ?

Reply:
Yes :+1: